client/selfrepair/tests/test_normandy_driver.js

```diff
@@ -36,21 +39,106 @@ describe('Normandy Driver', () => {
     });
   });
 
+  describe('saveHeartbeatFlow', () => {
+    const driver = new NormandyDriver();
+
+    afterEach(() => {
+      fetchMock.restore();
```

Comment:
Hmmm I think it actually makes more sense in `afterEach`. Moving it to `beforeEach` would look weird because we still want to call `fetchMock.get(url)` there.  (In the other example you linked to, it didn't matter as much because our `fetchMock.get` calls varied for each test and had to be defined separately in each `it` block.)

I'll switch the other test case to also use `restore()` in `afterEach`.